bindings/node.js/test/kzg.test.ts

```diff
@@ -64,8 +64,40 @@ const proofBadLength = randomBytes(BYTES_PER_PROOF - 1);
 const fieldElementValidLength = randomBytes(BYTES_PER_FIELD_ELEMENT);
 const fieldElementBadLength = randomBytes(BYTES_PER_FIELD_ELEMENT - 1);
 
-function bytesFromHex(hexString: string): Buffer {
-  return Buffer.from(hexString.slice(2), "hex");
+function bytesEqual(a: Uint8Array | Buffer, b: Uint8Array | Buffer): boolean {
+  if (a.length !== b.length) {
+    return false;
+  }
+  for (let i = 0; i < a.length; i++) {
+    if (a[i] !== b[i]) return false;
+  }
+  return true;
+}
+
+function padToEven(a: string): string {
+  return a.length % 2 ? `0${a}` : a;
+}
+
+function bytesFromHex(hex: string): Uint8Array {
+  if (typeof hex !== "string") {
+    throw new Error(`hex argument type ${typeof hex} must be of type string`);
+  }
+
+  if (hex.startsWith("0x")) {
+    hex = hex.slice(2);
+  }
+
+  if (hex.length % 2 !== 0) {
+    hex = padToEven(hex);
+  }
+
+  const byteLen = hex.length / 2;
+  const bytes = new Uint8Array(byteLen);
+  for (let i = 0; i < byteLen; i++) {
+    const byte = parseInt(hex.slice(i * 2, (i + 1) * 2), 16);
+    bytes[i] = byte;
+  }
+  return bytes;
```

Comment:
Alternatively, I'd suggest doing this which I believe is a lot simpler:

```js
function bytesFromHex(hexString: string): Uint8Array {
  const buffer = Buffer.from(hexString.slice(2), "hex");
  return new Uint8Array(buffer.buffer, buffer.byteOffset, buffer.byteLength);
}
```

Reply:
I initially thought of this but decided against this in case the c-kzg becomes browser compatible, since this is a test file i don't really have strong opinion so can update it accordingly

Reply:
can also 

```suggestion
function bytesFromHex(hexString: string): Uint8Array {
  return Uint8Array.from(Buffer.from(hexString.slice(2), "hex"))
}
```

Reply:
> decided against this in case the c-kzg becomes browser compatible

I'm not that experienced with javascript, why would this make a difference?

Reply:
@matthewkeil ah yes, that is even cleaner 👍

Reply:
Because buffer is not available in browser and if you would run these tests in karma (to always maintain browser compatibility)
but that can be done when c-kzg  gets at it i guess.